### DISCUSSION TITLE
dde-file-manager: Drag and drop in thumbnails/tabs

### DISCUSSION THREAD

**Discussion by benben962** [linuxdeepin/developer-center#3038] [Features Request & Ideas | 特性请求 & 头脑风暴]
The title is pretty straightforward but... This is a feature request to be able to drag and drop files from one folder to another when I open the two folders in a thumbnail. This is a feature in all the other file managers I know on GNU/Linux OS. If it is already a feature in the program, it is not on in the last version of the package in the Arch Linux repositories.

All best.

**Comment by BLumia** [comment]
Sorry my English is bad, what does 'open the two folders in a thumbnail' means? Did you means drag file(s) on a folder's thumbnail icon to move file into that folder? If do, that should works and maybe you got a bug.

**Comment by benben962** [comment]
Yes, it might be that I have a bug: 

![here is a link for a gif of the problem](https://zupimages.net/up/20/05/ytd5.gif).

When I try to drag the "Test.txt" to the "Vidéos" folder from the "Bureau" folder, I can't do it and I get a "forbidden" icon beside my cursor.

Don't apologize for your English, my sentence was not very clear and I should have added a gif in the first place (^_^')

**Comment by BLumia** [comment]
Oh I got what you means now. Drag'n'drop onto another tab is currently not supported, maybe we can add this feature in future updates, thanks for the feedback :)
